x-pack/plugins/snapshot_restore/public/application/sections/home/snapshot_list/snapshot_list.tsx

```diff
@@ -290,41 +257,41 @@ export const SnapshotList: React.FunctionComponent<RouteComponentProps<MatchPara
       />
     );
   } else {
-    const repositoryErrorsWarning = Object.keys(errors).length ? (
-      <Fragment>
-        <EuiCallOut
-          title={
-            <FormattedMessage
-              id="xpack.snapshotRestore.repositoryWarningTitle"
-              defaultMessage="Some repositories contain errors"
-            />
-          }
-          color="warning"
-          iconType="alert"
-        >
-          <FormattedMessage
-            id="xpack.snapshotRestore.repositoryWarningDescription"
-            defaultMessage="Snapshots might load slowly. Go to {repositoryLink} to fix the errors."
-            values={{
-              repositoryLink: (
-                <EuiLink {...reactRouterNavigate(history, linkToRepositories())}>
-                  <FormattedMessage
-                    id="xpack.snapshotRestore.repositoryWarningLinkText"
-                    defaultMessage="Repositories"
-                  />
-                </EuiLink>
-              ),
-            }}
-          />
-        </EuiCallOut>
-        <EuiSpacer />
-      </Fragment>
-    ) : null;
-
     content = (
-      <Fragment>
-        {repositoryErrorsWarning}
-
+      <>
+        {snapshots.length === SNAPSHOT_LIST_MAX_SIZE && (
+          <>
+            <EuiCallOut
+              color="warning"
+              iconType="help"
+              data-test-subj="maxSnapshotsWarning"
+              title={i18n.translate(
+                'xpack.snapshotRestore.snapshotsList.maxSnapshotsDisplayedTitle',
+                { defaultMessage: 'Snapshot list may be incomplete' }
```

Comment:
```suggestion
                { defaultMessage: 'Cannot show the full list of snapshots' }
```

Reply:
I feel like this text is a little more descriptive. It's not that the list is incomplete, it's that we just can't show all of the results.